driver/postgresql/driver.go

```diff
@@ -40,8 +41,16 @@ type Conn struct {
 
 // HasTable check if the schema has the migration table.
 func (c *Conn) HasTable() (bool, error) {
+	pgSchema := "public"
+	pgTable := c.table
+	if strings.Contains(pgTable, ".") {
```

Comment:
I think you can split everytime and count the number of elements in the name to get what to do. For example:

```
parts := strings.Split(c.Table, ".")
if len(parts) == 2 {
} else {
}
```

Also, what if there is a dot in the table name, as I believe this is allowed? What if the user specifies the full name of the table (database, schema, table)?

Reply:
That can happen, but such a user is going out of their way to avoid normal SQL in any DB. A caveat in the README could explain that table names with dots in them are not supported.

If you'll remember, an earlier PR made SCHEMA an explicit feature of the configuration, which you declined. There is no fool-proof way to support schemas without the explicit version.

At present, Rambler is promotes an anti-pattern in PG: use of public. https://wiki.postgresql.org/wiki/Database_Schema_Recommendations_for_an_Application

Reply:
Either a documentation warning about what is expected or handling all the possibilities in the code is fine either way for me.

Your earlier PR was overreaching in what it was modifying, and you've been overreacting to my comment on it.

 Rambler itself doesn't promote anything, and you can actually use multiple schemas in migrations already. Something not doing things the way you want them isn't a reason to use terms like "promotes an anti-pattern".

Reply:
I'm not overreaching nor overreacting. Because Rambler treats PG like MySQL, it forces people into the anti-pattern. That's why I included the link. It's easily fixed. As it stands this PR would do that.

I'm happy to help with Rambler. Are you willing to get help?